spec/cases/store_attribute_spec.rb

```diff
@@ -319,11 +319,25 @@
       expect(user.reload.price).to be 99
     end
 
-    it "should not include empty changes" do
+    it "should not mark attributes as dirty after reading original values" do
       reloaded_user = User.take
       reloaded_user.inspect
 
       expect(reloaded_user.changes).to eq({})
+      expect(reloaded_user.changed_attributes).to eq({})
+      expect(reloaded_user.changed?).to be false
+    end
+
+    it "should not mark attributes as dirty after default values are set" do
```

Comment:
Rails adds defaults to changes when they're first set (I tried with non-store attributes here https://github.com/palkan/store_attribute/pull/24)